specification/security/resource-manager/Microsoft.Security/preview/2023-02-01-preview/securityConnectors.json

```diff
@@ -416,6 +418,19 @@
           "type": "object",
           "description": "The AWS account's organizational data",
           "$ref": "#/definitions/AwsOrganizationalData"
+        },
+        "regions": {
+          "type": "array",
+          "description": "list of regions to scan",
+          "items": {
+            "type": "string",
+            "description": "aws regions names"
+          }
+        },
+        "accountName": {
```

Comment:
no org name for aws?

Reply:
No - it's management account 
same as member account